src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -442,7 +442,8 @@ println(results[1]);
 // will print "task2"
 ----
 
-NOTE: Please note that as discussed with workflow variables, results will be aggregated according to tasks termination. Consequently, the order in which values are put in the `results` variable depends of the order in which task1 and task2 finish their execution, especially if they are executed in parallel.
+NOTE: results will be aggregated according to the order declared in the dependency list. Consequently, if the "depends" tag of task3 contains the list [task1, task2], then results[0] will contain the result of task1 and results[1] the result of task2. If the depends list is [task2, task1], then results[0] will contain the result of task2 and results[1] the result of task1.
```

Comment:
With the studio it is confusing, I made the test. 
In general, it is the left to right order, but if you use the automatic layout button, as this orders the tasks from left to right in alphabetic order, it can display a visual order which does not corresponds to the actual XML order. It should be considered as a bug.